.github/workflows/ci.yml

```diff
@@ -0,0 +1,514 @@
+name: CI
```

Comment:
Did you intend to bring over the GHA CI as part of this pick?

Reply:
no i messed that up